.pipeline/checkstyle-suppressions.xml

```diff
@@ -6,6 +6,9 @@
 
 <suppressions>
   <!-- Suppress generated clients -->
-  <suppress files="[/\\]core/client[/\\]" checks=".*"/>
-  <suppress files="[/\\]orchestration/client[/\\]" checks=".*"/>
```

Comment:
I presume the normal slashes work fine on windows then 😄

Reply:
yeah ^^

I also assumed this was a windows workaround. I've checked, it's not necessary.

Reply:
(It didn't work)